.yaspellerrc.json

```diff
@@ -0,0 +1,152 @@
+{
```

Comment:
note how the config file which includes the dictionary is 44 lines shorter than the old dictionary.